### PR TITLE
disable computing related lessons

### PR DESCRIPTION
**Pull request by davidsbailey**
Finishes [PLAT-630]. Skips computing related lessons and always returns an empty list. Because the problem in the bug report is a server error which prevents the lesson edit page from loading, we can't simply disable the related lessons in the lesson edit page ui.

## Testing story

* Existing test coverage ensures that the related lessons feature continues to mostly work, even though it is not longer used in the UI
* I was not able to locally repro the problem shown in the bug report. However, since this code change clearly eliminates calling into the codepath in which the problem occurs, it seems like there is a good chance it will fix the reported issue.
* even though i could not repro the problem, I was able to manually verify that the "related lessons" section no longer appears on the lesson edit page.

## Future work

I opened [PLAT-646] to track re-enabling of this feature.

## Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

[PLAT-630]: https://codedotorg.atlassian.net/browse/PLAT-630

[PLAT-646]: https://codedotorg.atlassian.net/browse/PLAT-646

### CODE REVIEW

File: dashboard/app/controllers/lessons_controller.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def show
   # GET /lessons/1/edit
   def edit
     @lesson_data = @lesson.summarize_for_lesson_edit
-    @related_lessons = @lesson.summarize_related_lessons
+    @related_lessons = []
```
Comment (bethanyaconnor) [inline dashboard/app/controllers/lessons_controller.rb:29]: nit: can you add a comment explaining why this is just an empty list?

**Review by bethanyaconnor — APPROVED** [review]
Small nit but otherwise looks good!

File: dashboard/app/controllers/lessons_controller.rb (hunk 1 above)
Comment (davidsbailey) [reply dashboard/app/controllers/lessons_controller.rb:29]: I've added a brief comment. Please let me know if you'd like to see more.

Comment (bethanyaconnor) [reply dashboard/app/controllers/lessons_controller.rb:29]: Looks good! Thanks!

**Pull request merged**